test/suites/ipv6/suite_test.go

```diff
@@ -94,4 +100,58 @@ var _ = Describe("IPv6", func() {
 		})
 		Expect(internalIPv6Addrs).To(HaveLen(1))
 	})
+	It("should provision a static IPv6 prefix with node launch and set IPv6 as primary in the primary network interface", func() {
+		clusterDNSAddr := env.ExpectIPv6ClusterDNS()
+		nodeClass.Spec.Kubelet = &v1.KubeletConfiguration{ClusterDNS: []string{clusterDNSAddr}}
+		Expect(disableVPCCNIProvisioning(true)).ToNot(HaveOccurred())
+		pod := coretest.Pod()
+		env.ExpectCreated(pod, nodeClass, nodePool)
+		env.EventuallyExpectHealthy(pod)
+		env.ExpectCreatedNodeCount("==", 1)
+		output, err := env.EC2API.DescribeInstances(env.Context, &ec2.DescribeInstancesInput{
+			InstanceIds: []string{coretest.Pod().Spec.NodeName},
+		})
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(output.Reservations).To(BeNumerically(">=", 1))
```

Comment:
```suggestion
		Expect(output.Reservations).To(BeNumerically("==", 1))
```

We just expect a single reservation and a single instance, right?